src/pages/Admin/Charity/Nav.tsx

```diff
@@ -21,13 +21,9 @@ export default function Nav() {
       <NavLink end to={adminRoutes.proposals} className={styler}>
         Proposals
       </NavLink>
-      {/*      <NavLink
-        end
-        to={`${adminRoutes.widget_config}/${endowmentId}`}
-        className={styler}
-      >
-        Embed Widget
-      </NavLink>*/}
+      <NavLink end to={`${adminRoutes.widget_config}`} className={styler}>
```

Comment:
The endowment should be navigated to their own widget config page, i.e. their endowment should be selected by default in [EndowmentCombobox](https://github.com/AngelProtocolFinance/angelprotocol-web-app/tree/master/src/pages/Admin/Charity/WidgetConfigurer/WidgetUrlGenerator/EndowmentCombobox) when the page loads, right @SovereignAndrey ?

If so, the target URL should be changed to the one that was commented out above:
```tsx
const { endowmentId } = useAdminResources();
...
<NavLink end to={`${adminRoutes.widget_config}/${endowmentId}`} className={styler}>
```

Reply:
> The endowment should be navigated to their own widget config page, i.e. their endowment should be selected by default in [EndowmentCombobox](https://github.com/AngelProtocolFinance/angelprotocol-web-app/tree/master/src/pages/Admin/Charity/WidgetConfigurer/WidgetUrlGenerator/EndowmentCombobox) when the page loads, right @SovereignAndrey ?
> 

Correct! @abbasali1994, this is the way we need it for that admin link.

Reply:
Understood , I will make the following change